filebeat/docs/migration.asciidoc

```diff
@@ -361,11 +361,11 @@ Forwarder.
 The custom fields (added from the configuration file) are set as top-level
 fields in Logstash Forwarder but are grouped together under a `fields`
 dictionary in Filebeat. If you need the old behavior during the migration phase,
-you can use the <<fields-under-root>> configuration option:
+you can use the <<fields-under-root-log>> configuration option:
 
 [source,yaml]
 -------------------------------------------------------------------------------------
-filebeat.prospectors:
+filebeat.inputs:
 - type: log
   paths:
     - /var/log/*.log
```

Comment:
Should we use the asciidoc variable for all the `filebeat` reference in this file? ({beatname_uc})

Reply:
Yes...not urgent, but over I'm trying to replace references to product names with asciidoc variables whenever I open up a file